common-util/AbiAndAddresses/index.jsx

```diff
@@ -2,3 +2,7 @@ export * from './mintNftContractGoerli';
 export * from './mintNftContractMainnet';
 export * from './veOlasContract';
 export * from './delegateContributeContract';
+export * from './contributeManager';
+export * from './stakingToken';
+export * from './olas';
+export * from './genericErc20';
```

Comment:
What's the difference between `olas` and `genericErc20`? I would say `olas` is enough

Reply:
`olas` is based on generic ERC20

Reply:
aaaah didn't know that, copy-pasted from registry, good point, will update